jacodb-panda-dynamic/src/main/kotlin/org/jacodb/panda/dynamic/ets/dto/convert.kt

```diff
@@ -0,0 +1,712 @@
+                }
+            }
+
+            is LocalDto -> EtsLocal(
+                name = value.name,
+                type = convertToEtsType(value.type),
+            )
+
+            is ConstantDto -> convertToEtsConstant(value)
+
+            is NewExprDto -> EtsNewExpr(
+                type = convertToEtsType(value.type) // as ClassType
+            )
+
+            is NewArrayExprDto -> EtsNewArrayExpr(
+                elementType = convertToEtsType(value.type),
+                size = convertToEtsEntity(value.size),
+            )
+
+            is DeleteExprDto -> EtsDeleteExpr(
+                arg = convertToEtsEntity(value.arg)
+            )
+
+            is TypeOfExprDto -> EtsTypeOfExpr(
+                arg = convertToEtsEntity(value.arg)
+            )
+
+            is InstanceOfExprDto -> EtsInstanceOfExpr(
+                arg = convertToEtsEntity(value.arg),
+                checkType = convertToEtsType(value.checkType),
+            )
+
+            is LengthExprDto -> EtsLengthExpr(
+                arg = convertToEtsEntity(value.arg)
+            )
+
+            is CastExprDto -> EtsCastExpr(
+                arg = convertToEtsEntity(value.arg),
+                type = convertToEtsType(value.type),
+            )
+
+            is PhiExprDto -> EtsPhiExpr(
+                args = value.args.map { convertToEtsEntity(it) },
+                argToBlock = emptyMap(), // TODO
+                type = convertToEtsType(value.type),
+            )
+
+            is ArrayLiteralDto -> EtsArrayLiteral(
+                elements = value.elements.map { convertToEtsEntity(it) },
+                type = convertToEtsType(value.type), // TODO: as EtsArrayType,
+            )
+
+            is ObjectLiteralDto -> EtsObjectLiteral(
+                properties = emptyList(), // TODO
+                type = convertToEtsType(value.type),
+            )
+
+            is UnaryOperationDto -> EtsUnaryOperation(
+                op = convertToEtsUnaryOp(value.op),
+                arg = convertToEtsEntity(value.arg),
+            )
+
+            is BinaryOperationDto -> EtsBinaryOperation(
+                op = convertToEtsBinaryOp(value.op),
+                left = convertToEtsEntity(value.left),
+                right = convertToEtsEntity(value.right),
+            )
+
+            is RelationOperationDto -> EtsRelationOperation(
+                relop = value.op,
+                left = convertToEtsEntity(value.left),
+                right = convertToEtsEntity(value.right),
+            )
+
+            is InstanceCallExprDto -> EtsInstanceCallExpr(
+                instance = convertToEtsEntity(value.instance),
+                method = convertToEtsMethodSignature(value.method),
+                args = value.args.map {
+                    val etsEntity = convertToEtsEntity(it)
+                    if (etsEntity is EtsValue) return@map etsEntity
+                    val newLocal = EtsLocal("_tmp${freeLocal++}", EtsUnknownType)
```

Comment:
Please, use empty lines to separate different code blocks

Reply:
This is a single code block.